src/openrct2/CmdlineSprite.cpp

```diff
@@ -50,30 +50,20 @@ assert_struct_size(rct_sprite_file_header, 8);
 
 struct SpriteFile
 {
+public:
     rct_sprite_file_header Header;
     std::vector<rct_g1_element> Entries;
     std::vector<uint8_t> Data;
+    void AddImage(ImageImporter::ImportResult& image);
+    bool Save(const utf8* path);
+    static std::optional<SpriteFile> Open(const utf8* path);
+
+private:
     bool isAbsolute = false;
     void MakeEntriesAbsolute();
     void MakeEntriesRelative();
-    bool Save(const utf8* path);
-    static std::optional<SpriteFile> Open(const utf8* path);
 };
 
-#ifdef _WIN32
-
-static FILE* fopen_utf8(const char* path, const char* mode)
-{
-    auto pathW = String::ToWideChar(path);
-    auto modeW = String::ToWideChar(mode);
-    auto file = _wfopen(pathW.c_str(), modeW.c_str());
-    return file;
-}
-
-#    define fopen fopen_utf8
-
-#endif
-
```

Comment:
(On commit https://github.com/OpenRCT2/OpenRCT2/pull/13590/commits/8a91330eb7df1e56de3ceddce95937d94396ce82)

Not sure this became useless on previous commits or if it was never touched in this PR at all, but this cleanup seems a bit lost here.

Reply:
It could have been removed after the second file stream commit